app/config.py

```diff
@@ -0,0 +1,80 @@
+import json
+import os
+
+from kombu import Exchange, Queue
+
+
+class QueueNames:
+    LETTERS = 'letter-tasks'
+    SANITISE_LETTERS = 'sanitise-letter-tasks'
+
+
+class TaskNames:
+    PROCESS_SANITISED_LETTER = 'process-sanitised-letter'
+    UPDATE_BILLABLE_UNITS_FOR_LETTER = 'update-billable-units-for-letter'
+    UPDATE_VALIDATION_FAILED_FOR_TEMPLATED_LETTER = 'update-validation-failed-for-templated-letter'
+
+
+class Config:
+    NOTIFY_ENVIRONMENT = os.environ['NOTIFY_ENVIRONMENT']
+
+    AWS_REGION = 'eu-west-1'
+    TEMPLATE_PREVIEW_INTERNAL_SECRETS = json.loads(
+        os.environ.get('TEMPLATE_PREVIEW_INTERNAL_SECRETS', '[]')
+    )
+    NOTIFY_APP_NAME = 'template-preview'
+    DANGEROUS_SALT = os.environ.get('DANGEROUS_SALT')
+    SECRET_KEY = os.environ.get('SECRET_KEY')
+
+    CELERY = {
+        'broker_url': 'sqs://',
+        'broker_transport_options': {
+            'region': AWS_REGION,
+            'visibility_timeout': 310,
+            'wait_time_seconds': 20,  # enable long polling, with a wait time of 20 seconds
+            'queue_name_prefix': ({
+                'test': 'test',
+                'development': os.environ.get('NOTIFICATION_QUEUE_PREFIX', 'development'),
+                'preview': 'preview',
+                'staging': 'staging',
+                'production': 'live',
+            }[NOTIFY_ENVIRONMENT]),
+        },
+        'timezone': 'Europe/London',
+        'worker_max_memory_per_child': 50,
+        'imports': ['app.celery.tasks'],
+        'task_queues': [
+            Queue(QueueNames.SANITISE_LETTERS, Exchange('default'), routing_key=QueueNames.SANITISE_LETTERS)
+        ],
+    }
+
+    # if we use .get() for cases that it is not setup
+    # it will still create the config key with None value causing
+    # logging initialization in utils to fail
+    if 'NOTIFY_LOG_PATH' in os.environ:
+        NOTIFY_LOG_PATH = os.environ.get('NOTIFY_LOG_PATH')
+
+    EXPIRE_CACHE_IN_SECONDS = 600
+
+    if os.environ.get('STATSD_ENABLED') == "1":
+        STATSD_ENABLED = True
+        STATSD_HOST = os.environ.get('STATSD_HOST')
+        STATSD_PORT = 8125
+    else:
+        STATSD_ENABLED = False
```

Comment:
it might be nice to replicate the existing other app configs here and just have this without the if statement if that works

```
    STATSD_HOST = os.getenv('STATSD_HOST')
    STATSD_PORT = 8125
    STATSD_ENABLED = bool(STATSD_HOST)
```

Reply:
Sure, changed in https://github.com/alphagov/notifications-template-preview/pull/651/commits/d3a5ab9d87c87d9df7ce4b17e5d1be26f25c9634.